tck/src/main/java/org/eclipse/configjsr/CDIPlainInjectionTest.java

```diff
@@ -46,6 +40,13 @@
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.Test;
 
+import static org.eclipse.configjsr.matchers.AdditionalMatchers.floatCloseTo;
```

Comment:
are those imports really in use?

Reply:
Reformatting of imports, static now below normal